packages/events/__tests__/ResponderEventPlugin-test.internal.js

```diff
@@ -1360,4 +1360,88 @@ describe('ResponderEventPlugin', () => {
     run(config, three, nativeEvent);
     expect(ResponderEventPlugin._getResponder()).toBe(null);
   });
+
+  it('should determine the first common ancestor correctly', () => {
```

Comment:
Since we're not using Responder plugin it's worth adding a comment explaining *why* the test is here and how it's related to Resonder.